### DISCUSSION TITLE
Does Rowy use custom claims

### DISCUSSION THREAD

**Discussion by kavinmehta** [AntlerVC/firetable#1159] [Support / Q&A] [answered]
Hi,

Do you use custom claims in firebase auth for setting up of roles or all roles are from the '/_rowy_/userManagement/users/<uid>/roles'  document?

Thanks!

**Comment by BeeBombshell** [comment, marked as answer]
Hey, @kavinmehta. Rowy does use custom claims in Firebase Auth to define and manage user roles, rather than relying solely on the `/rowy/userManagement/users//roles` document. You can add a new role to your workspace via the **Workspace settings**. Click on **Invite User**. Under the **Roles** section in the modal, click on **Add new role**. (Refer to the screenshot below)

Here's a link to our docs for more information on setting up custom roles: https://docs.rowy.io/how-to/roles.

<img width="1512" alt="image" src="https://user-images.githubusercontent.com/53828745/224964195-2a73a36d-1fe2-4315-a2df-c2ffecf18d63.png">
